### PR TITLE
update some system variable

### PR DESCRIPTION
**Pull request by xiongjiwei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -205,6 +230,10 @@ mysql> SELECT * FROM t1;
 - 单位：秒
 - 该变量表示交互式用户会话的空闲超时。交互式用户会话是指使用 `CLIENT_INTERACTIVE` 选项调用 [`mysql_real_connect()`](https://dev.mysql.com/doc/c-api/5.7/en/mysql-real-connect.html) API 建立的会话（例如：MySQL shell 客户端）。该变量与 MySQL 完全兼容。
 
+### `last_insert_id` <span class="version-mark">从 v5.3.0 版本开始引入</span>
+
+- 返回 `LAST_INSERT_ID()` 的结果。与 MySQL 中的 `last_insert_id` 一致。
```
Comment (TomShawn) [inline system-variables.md:235]: 是否有作用域呢？或者是否为只读变量呢？

File: system-variables.md [hunk 2]
```diff
@@ -180,6 +201,10 @@ mysql> SELECT * FROM t1;
 - 默认值：（系统主机名）
 - 这个变量一个只读变量，表示 TiDB server 的主机名。
 
+### `identity` <span class="version-mark">从 v5.3.0 版本开始引入</span>
+
+- `last_insert_id` 的别名。
```
Comment (TomShawn) [inline system-variables.md:206]:
```suggestion
- 该变量为变量 [`last_insert_id`](#last_insert_id-从-v530-版本开始引入) 的别名。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**